src/lib/polygon/PolygonUtils.js

```diff
@@ -0,0 +1,41 @@
+/* global PolygonConstants */
+
+class PolygonUtils { // eslint-disable-line no-unused-vars
+    /**
+     * @param {number} coins USDC coins in decimal
+     * @returns {number} Number of Cents
+     */
+    static coinsToCents(coins) {
```

Comment:
I guess this comment follow the previous one about cents. We can keep it like that even if it's not exactly what's happenning (since it's not cents but 1/10000 of a cent iirc) or change it for consistency to something like weiToCoins or usdcweiToCoins or anything else more correct than cents.